datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -472,6 +473,13 @@ CREATE TABLE dcsa_im_v3_0.equipment (
 
 -- Supporting FK constraints
 CREATE INDEX ON dcsa_im_v3_0.equipment (iso_equipment_code);
+CREATE INDEX ON dcsa_im_v3_0.equipment (equipment_reference);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.requested_equipment_equipment CASCADE;
```

Comment:
requested_equipment_equipment is not a table in our DB

Reply:
Sorry, that was a naming mistake.